cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -271,6 +271,7 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 	// update the cstorvolume reference, phase as "Bound" and desired
 	// capacity
 	cvc.Spec.CStorVolumeRef = volumeRef
+	cvc.Spec.Policy = volumePolicy.Spec
```

Comment:
(not just related to this PR) we are modifying the objects from store.. is this fine?

Reply:
Yes, that's k8s does that for there resources to have some default values